controllers/busRouteAssignController.js

```diff
@@ -0,0 +1,28 @@
+import { success, fail } from '../function/respond';
+import { buses as busModel, routes as routeModel } from '../models';
+
+
+export const Assign = async (req, res) => {
+	const { plate, code } = req.params;
```

Comment:
I can see that we don't have any validation here and also this function may crush the server you may use try and catch that may fix it and validate all these inputs with Joi.

Reply:
@Delyc  did you work on this feedback?

Reply:
no yet done, working on it rn